### PR TITLE
Changes texture request to default to a correct request to thor as of yesterday evening

### PR DESCRIPTION
**Pull request by Rovanion**
Now you get data again!

### CODE REVIEW

File: src/weather_magic/textures.cljs [hunk 1]
```diff
@@ -44,19 +44,27 @@
...
 (defn load-data
-  "Loads data from thor into a texture. Returs a map with {:key
-  str :map texture-map} where :key holds how to find the newly loaded
-  texture in texture-map."
+  "Loads data from thor into a texture.
+
+  There are two optional associative arguments:
+
+  :variable       - The type of data to request from the backend,
+                    normally 'temperature' or 'percipitation'.
```
Comment (yotomten) [inline src/weather_magic/textures.cljs]: *precipitation

**Comment by yotomten** [comment]
@Rovanion Rovanion requested a deployment to weather-front-pr-80 23 minutes ago Pending.

Why is it pending?

File: src/weather_magic/textures.cljs (hunk 1 above)
Comment (yotomten) [reply src/weather_magic/textures.cljs]: Quite important in this case ;) ;)

**Review by yotomten — CHANGES_REQUESTED** [review]
Correct spelling

**Review by yotomten — APPROVED** [review]

**Comment by Rovanion** [comment]
It's pending because I'm only allowed to host five free dynos on Heroku. That means only three PR dynos because I got one other project on there plus the master weather-front dyno.

**Pull request merged**